src/filtering/FuncOperations.js

```diff
@@ -67,7 +67,7 @@ export const nameStringOperation = (op, value) => {
   const strOp = stringOperation(op, value);
   return (fieldValue, card) => {
     let expandedValue = fieldValue
-      .replace(new RegExp(card.details.name, 'g'), NAME_PLACEHOLDER)
+      .replace(new RegExp(card.details.name.replace('+', '\\+'), 'g'), NAME_PLACEHOLDER)
```

Comment:
```suggestion
      .replace(new RegExp(card.details.name.replace(/+/g, '\\+'), 'g'), NAME_PLACEHOLDER)
```

Reply:
by the way, this all is the result of Node not having a proper `replaceAll` function until v15, which is just dumb.